ansible/monitoring.yml

```diff
@@ -15,6 +15,55 @@
         tasks_from: config.yml
       tags: config
 
+    - name: Define tmp directories on tmpfs
```

Comment:
Not in reference to this line in particular, but would this could fit better in the podman role?

Reply:
The problem is that the user we're running podman as isn't defined in the role, only at the appliance level. I agree it feels like this (and the validate and the `podman_tmp_dir_root`) should all really be in the podman role, so if you can see a way of achieving that let me know.